### DISCUSSION TITLE
Thoughts - Exporting & Inputs

### DISCUSSION THREAD

**Discussion by thejackiechanuk** [Ayagikei/LifeUp#138] [Ideas]
Hello,

I have no experience with app development, so what follows might be complete garbage.

Would it be possible to send information input to LifeUp via NFC tags or QR tags?

And would it be possible to export my profile; tasks, achievements, items, etc  as a template for someone else?
I, somewhat managed to test, by back-up and restore but I don't have duplicate LifeUp account to verify this.

Or even share tasks, though it'll be tricky with people customising different stats or if there's an non-exiting item reward... Or how people will set up their experience gains.

Many thanks!

**Comment by Ayagikei** [comment]
Hi, thanks for the discussion!

You can send messages via NFC or QR tags - it's based on the API functionality provided by LifeUp.

Due to limited space, you may need to consult our API documentation for details:
1. [https://docs.lifeupapp.fun/en/index.html#/guide/api](https://docs.lifeupapp.fun/en/index.html#/guide/api)
2. [https://docs.lifeupapp.fun/en/index.html#/guide/api_cloud](https://docs.lifeupapp.fun/en/index.html#/guide/api_cloud)

Some additional information:
1. You need to refer to the API documentation to write the appropriate API URL to call LifeUp (enter information).
2. For NFC tags, you may need to use automated tools such as Tasker and Macrodroid to scan NFC tags. LifeUp does not provide native NFC support.
3. For QR tags, you can generally try to use the system browser or camera to scan. If these don't apply, our LifeUp Cloud extension app also provides scanning functionality.

---

You can use the backup and restore functions to share complete data templates.

But it is full data, not incremental, which means it will cover all the user's data.

We've seen some community users share their templates using this method.
